.env.sample

```diff
@@ -91,6 +91,10 @@ DATABASE_URL="postgresql://sn:password@db:5432/stackernews?schema=public"
 # FOR DOCKER ONLY #
 ###################
 
+# containers can't use localhost, so we need to use the container name
+IMGPROXY_URL_DOCKER=http://imgproxy:8080
+MEDIA_URL_DOCKER=http://s3:4566/uploads
+
```

Comment:
I spent the better part of 10 hours trying to avoid this 😮‍💨 and all the related changes.

Things that didn't work:

- any kind of normal docker compose configuration that uses the default network
- running a reverse proxy container
    - I almost got it working with `imgproxy.localhost` and `s3.localhost` and docker compose `links` but after I got that all figured out, I learned macos doesn't support localhost subdomains
- intercepting fetches in the service worker transforming domains, e.g. `imgproxy:3001` -> `localhost:3001`

... I'm not sure what else I could have tried. We more or less need the host, app, worker, s3, and imgproxy to all share some kind of name resolution service. This is easy enough to configure in containers, but we can't reliably alter (nor should we) the host's dns.